data/successors/successor side missions.txt

```diff
@@ -0,0 +1,488 @@
+			label accept
+			`	"Thou has my deep and certain thanks. Tell none, but the High Houses are beastly to us academics." The biologist, who introduces themselves as "Kaatiora", loads their collection of equipment into your ship---though not without almost dropping one particularly fragile-looking item---and is eventually ready to leave.`
+				accept
+	on stopover
+		conversation
+			`On the way here, Kaatiora has been impressively inquisitive, continually asking you questions about human biology with a level of verve and excitement you though previously impossible. Once they exhausted your knowledge of every detail of human biology, they moved on without hesitation to every other Earth-related species they knew of from Predecessor records, from cattle to frogs to far more exotic ones that you've never heard of in your life but apparently are all the rage among the circles of Successor xenobiology.`
+			`	It is somewhat relieving when you finally arrive at the planet and Kaatiora momentarily departs, once again carrying their more-than-armful of equipment and excitedly but carefully scanning the volcanic hot-water pools and small lakes around your landing site, occasionally exclaiming with excitement.`
+			`	Eventually they return to your ship, satisfied with the amount of data they have collected, but still obviously energized by the experience.`
+			`	Before leaving, you realize this would be a good time to learn more about the Successors from an expert. Besides, since you spent the majority of the time on the way here answering Kaatiora's questions, you are probably entitled to ask a few of your own.`
+			choice
+				`	(Ask about the Successors and biology.)`
+					goto ask
+				`	(Don't ask them.)`
+					launch
+			label ask
+			`	As expected, Kaatiora is more than happy to share their biological knowledge, and asks if there is anything in particular you'd like to know about.`
+			choice
+				`	"What can you tell me about the Successors?"`
+					goto successor
+				`	"Were the Predecessors any different, biologically speaking, from the present Successors?"`
+					goto predecessor
+				`	"What is the life in this star cluster like?"`
+					goto cluster
+				`	"I have no more questions."`
+					launch
+			label successor
+			`	"Much of our evolutionary history, we spent beneath the waves of our home planet, and while now you clearly see we breathe the same air as you, we still retain our cold and water-loving tendencies. Our respiration works just as well submerged as on dry land, though we must retain some degree of moisture on our skin.`
+			`	Apart from that, we are a hermaphroditic species, like most from our home planet, and as we took to the stars, we learned that this is uncommon but not rare, and so there is some curiosity as to why our planet had such a diversity of such species while most do not. Perhaps, since we grew up under a harsh sun, the redundancy provided by our divorce from biological sex was beneficial to our evolution.`
```

Comment:
The first line may have too many commas?
```suggestion
			`	"Apart from that, we are a hermaphroditic species, like most from our home planet, and as we took to the stars, we learned that this is uncommon but not rare, and so there is some curiosity as to why our planet had such a diversity of such species while most do not. Perhaps, since we grew up under a harsh sun, the redundancy provided by our divorce from biological sex was beneficial to our evolution.`
```